.github/workflows/ci.yaml

```diff
@@ -160,6 +160,12 @@ jobs:
         mypy --version
         mypy tools/api.py
 
+  Licensing:
```

Comment:
Can you add steps to free up the space like other CI checks? I remember in past we had some problems with available free space and had to include that step to free up more space.

Reply:
can we make this lowercase?

Reply:
Sure!